src/main/java/sg/ncl/ConnectionProperties.java

```diff
@@ -331,6 +331,10 @@ public String saveImage() {
         return HTTP_MODE + sioAddress + ":" + sioPort + "/" + imageEndpoint;
     }
 
+    public String deleteImage(String teamId, String imageName) {
```

Comment:
Modify endpoint to deleteImage(String imageId)

/imageEndpoint/imageId
Should not be using /imageEndpoint/imageName/delete?teamId=teamId
teamId can be retrieve from the ImageEntity in the SIO instead